openedx/features/content_tagging/rules.py

```diff
@@ -35,27 +38,61 @@ def is_taxonomy_admin(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool
     return False
 
 
+def is_taxonomy_admin(user: User) -> bool:
+    """
+    Returns True if the given user is a Taxonomy Admin.
+
+    Taxonomy Admins include global staff and superusers.
+    """
+    return oel_tagging.is_taxonomy_admin(user)
+
+
+def is_taxonomy_user_for_org(user: User, org: Organization) -> bool:
+    """
+    Returns True if the given user is a Taxonomy User for the given org.
+
+    Taxonomy users include global staff and superusers, course creators who can create courses for any org and
+    content creators for this specific org.
+    """
+    return is_content_creator(user, org.short_name)
```

Comment:
As noted, we don't need any of this "is_content_creator" logic anymore. Only global staff + superusers can add/edit taxonomies, and anyone can view an enabled taxonomy, no matter its org.

Reply:
Done!